python/paddle/fluid/layers/control_flow.py

```diff
@@ -912,6 +913,71 @@ def _complete(self):
                    "is_test": self.is_test})
 
 
+def while_loop(cond, body, loop_vars, name=None):
+    """
+    This while_loop operator control. Repeat while_loop body until cond return False.
```

Comment:
"Repeat" -> "Repeats" (Because API is third-person)
"until cond return" -> "until `cond` returns"

Reply:
Done.